Gemfile

```diff
@@ -1,20 +1,23 @@
 source 'https://rubygems.org'
 
-ruby '2.6.3'
+ruby '2.7.2'
 
-gem 'jekyll', "3.8.6"
+gem "jekyll", '~> 4.2.0', github: "jekyll/jekyll"
 gem 'html-proofer'
 gem "rack", ">= 2.0.6"
 gem 'asciidoctor'
-gem 'asciidoctor-pdf', '~> 1.5.3'
```

Comment:
I believe we need this as it's used by `scripts/build_pdfs` and `scripts/build_pdfs_asciidoc.sh` ?

Reply:
@teesloane At this time, we don't need to use `asciidoctor-pdf` package as a local package.

Previously, the `scripts/build_pdfs` script uses `asciidoctor-pdf` of the local package. However, `scripts/build_pdfs_asciidoc.sh` expected to use `asciidoctor-pdf` of global package, and I updated `scripts/build_pdfs`  as the same way. `scripts/build_pdfs_asciidoc.sh` is expected to run on a circleci job, and the executor is `asciidoctor/docker-asciidoctor`, also  `asciidoctor-pdf` package has installed in its image.

Here is the change in the `scripts/build_pdfs`.
> https://github.com/circleci/circleci-docs/pull/5146/files#diff-a68ac2d75157c9152f1f6c45bc5b054eef76bd1cdc71500efdf56495e75e8397R14

So, to run these scripts locally, you can use the following command.

```
$ docker run -it --rm -v `pwd`:/tmp asciidoctor/docker-asciidoctor:1.1.0 bash -c 'cd /tmp && sh ./scripts/build_pdfs_asciidoc.sh'
```